contrib/cni/README.md

```diff
@@ -40,19 +45,19 @@ two plugins necessary for the example CNI configurations are `loopback` and
 
 #### CNI Plugin Installation From Source
 
-This tutorial will use the latest version of `CNI` plugins from the master branch and build it from source.
+This tutorial will use the latest version of `CNI` and build it from source.
 
 Download the `CNI` plugins source tree:
 
 ```bash
 git clone https://github.com/containernetworking/plugins
 cd plugins
-git checkout v0.8.1
+git checkout v0.8.7
```

Comment:
to be honest, I am not sure we test against this and we should probably check it works before changing it :upside_down_face:

Reply:
Well the main go.mod has this version :).